packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Inbound/XCA ITI-39 Inbound Interface/destinationConnector-AWS Lambda Interface-responseTransformer-step-0-Process response.js

```diff
@@ -32,4 +26,39 @@ try {
 } catch(ex) {
 	if (globalMap.containsKey('TEST_MODE')) logger.error('XCA ITI-39 Inbound Processor: ' + ex);
 	throw ex;
-}
\ No newline at end of file
+}
+
+// processing script
```

Comment:
its not postprocessing logic anymore though. It was postprocessing logic when there were 2 channels, and it was a postprocessor for the second channel. But now there is only 1 channel, and so the logic is in the correct place based on the order of events.

Reply:
I see, it makes sense.

Just to confirm: did you test this alternative flow, where we'd execute the code inside the `if` below? Wondering especially because the `responseMap.get(channelMap.get('DEST_ID'));` and the fact we moved this from another channel.

Reply:
When I tested, responseMap.get(channelMap.get('DEST_ID') was always empty and I could not figure out what edge case it was handling or get it to be triggered. Very possible the whole block can all be deleted.

Reply:
AFAIK, at least for outbound, it was working when there were two channels.

We gotta be sure we're not losing an important part of the logic, which is to build an OperationOutcome if there's an error while processing the message/channel.

I still think we should have a post processor logic for this. Do we have it (and thus we can remove this)?